File: AndroidRepairApp/RepairApp/app/src/main/java/com/canviz/repairapp/data/InspectionModel.java
package com.canviz.repairapp.data;

import org.json.JSONException;
import org.json.JSONObject;

public class InspectionModel {
    private final JSONObject mData;

    public InspectionModel(JSONObject data) {
        mData = data;
    }

    public int getId(){
        try{
            return mData.getInt("ID");
        }catch(JSONException e) {
            return 0;
        }
    }

    public String getDateTime(){
        try{
            return mData.getString("sl_datetime");
        }catch(JSONException e) {
            return null;
        }
    }

    public String getFinalized(){
        try{
            return mData.getString("sl_finalized");
        }catch(JSONException e) {
            return null;
        }
    }
}

File: XamarinRepairApp/XamarinRepairApp/Model/InspectionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Json;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

using XamarinRepairApp.Utils;

namespace XamarinRepairApp.Model
{
    public class InspectionModel
    {
        private JsonValue mData;

        public InspectionModel(JsonValue jsonValue)
        {
            mData = jsonValue;
        }

        public int GetId()
        {
            return Helper.GetInt(mData["ID"]);
        }

        public String GetDateTime()
        {
            return Helper.GetDateTime(mData["sl_datetime"]);
        }

        public String GetFinalized()
        {
            return Helper.GetDateTime(mData["sl_finalized"]);
        }

        public String GetInspector()
        {
            return Helper.GetString(mData["sl_inspector"]);
        }

        public String GetEmailaddress()
        {
            return Helper.GetString(mData["sl_emailaddress"]);
        }

    }
}